migrations/sqls/20190806200938-issue-202-up.sql

```diff
@@ -0,0 +1,41 @@
+/* Replace with your SQL commands */
+
+UPDATE services
+SET programId = (
+  SELECT id FROM programs WHERE programName = "Community Engagement and Organizing"
+)
+WHERE serviceName = "Leadership Development - Monthly Meetings";
+
+INSERT INTO services (serviceName, serviceDesc, programId)
+VALUES
+  ("General Consultation", "General Consultation",
+    (SELECT id FROM programs WHERE programName = "Immigration")
+  ),
+  ("Translation", "Translation", (
+    SELECT id FROM programs WHERE programName = "Immigration"
+  )),
+  ("Green Card Renewal", "Green Card Renewal", (
+    SELECT id FROM programs WHERE programName = "Immigration"
+  )),
+  ("I-821 Temporary Protected Status (TPS)", "I-821 Temporary Protected Status", (
```

Comment:
Should the second be "I-821 Temporary Protected Status (TPS)" also for consistency?

Reply:
Good catch, updated.